### PR TITLE
Added SearchTimeEntries functionality

### PR DESCRIPTION
**Pull request by prasanthchaduvula**
## Notion card
https://www.notion.so/6afc9500b2cc42af86be55f37894fb09?v=185bd45b9dc24dfcad9c57c07d32a7c0&p=208d11342190430c819fa5b593fb56d8&pm=s

## Summary

We can see other user’s name in the dropdown. By default it shows the current user and we can select other team members to view their time entries..

## Preview

https://www.loom.com/share/7e8a2365528242b6ab7e78716371e451

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

### Checklist:

- [x] I have manually tested all workflows
- [x] I have performed a self-review of my own code

### CODE REVIEW

**Comment by apoorv1316** [comment]
@prasanthchaduvula Did we change the designs? I can only see the old design where the dropdown is in the center of the page and also has a search bar.https://www.figma.com/file/4gdPDRPqGqQqPYsO3Qj8CG/Miru-App?node-id=838%3A7861

**Comment by prasanthchaduvula** [comment]
@apoorv1316 Yes. I spoke with Supriya and we decided to go with this UI. Currently UI redesigning is going on. Please test the functionality.

**Review by apoorv1316 — APPROVED** [review]

File: app/javascript/src/common/SearchTimeEntries.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import React from "react";
+
+import Select from "react-select";
+
+const SearchTimeEntries = ({
+  selectedEmployeeId,
+  setSelectedEmployeeId,
+  employeeList
+}) => {
+  const currentUser = employeeList?.find((emp) => emp.value == selectedEmployeeId);
```
Comment (sudeeptarlekar) [inline app/javascript/src/common/SearchTimeEntries.tsx:10]: nit: use `===` for comparison,

**Review by sudeeptarlekar — APPROVED** [review]
LGTM.

File: app/javascript/src/common/SearchTimeEntries.tsx (hunk 1 above)
Comment (prasanthchaduvula) [reply app/javascript/src/common/SearchTimeEntries.tsx:10]: @sudeeptarlekar The backend data is coming through API is type variant. So If we use strict equality `===` It wont display the current user by default.

**Pull request merged**